### ISSUE TITLE
[FR] Add FTUE

### ISSUE THREAD

**Issue by matmair** [inventree/InvenTree#2907]
**Problem**
We currently just drop the user in - some kind of out-of-the-box or first-time-user widget could be a good way to familiarize the user with the interface.

**Suggested solution**
Add a FTUE with:
- [ ] multi-step-wizards
- [ ] state saving per user -> https://github.com/inventree/InvenTree/pull/10715
- [ ] short introduction
- [ ] link to the docs
- [ ] link to the blog
- [ ] a way for plugins to provide pages

**Describe alternatives you've considered**
The current way works. For 1.0.0 maybe a basic setup would be nice

**Examples of other systems**
Nextcloud has a nice, 3 step FTUE - there are fuller examples in closed source systems but I am not sure if I am able to share infos about that.

**Do you want to develop this?**
Yes - When the time is ready

**Comment by SchrodingersGat** [comment]
Yeah, this would be a major step forward in terms of user friendliness. 100% on board

**Comment by matmair** [comment]
This will be a lot about user-friendliness - this does not need to be a core-dev activity once the code for #2284 and the first few tasks are done.

**Comment by matmair** [comment]
Waiting for #3901

**Comment by SchrodingersGat** [comment]
[wiki.js](https://js.wiki/) is an example of a good first-time-admin-experience
